src/frontend/src/lib/components/ui/ButtonHero.svelte

```diff
@@ -6,7 +6,7 @@
 	export let ariaLabel: string;
 </script>
 
-<Button on:click {ariaLabel} {disabled} {testId} colorStyle="tertiary" link paddingSmall>
+<Button on:click {ariaLabel} {disabled} {testId} colorStyle="tertiary" link>
```

Comment:
Why is the something related to the "Hero" pane modified in this PR?

Reply:
Because of the Button changes. Buttons do not match the figma at all at the moment. To match this Promocard to the figma design, the Button need adjustments to match the figma design.

Reply:
I'll make a separate PR

Reply:
I forgot to send the message, but yes, I would prefer another PR too, and if necessary for this, to be done first. It should be a quick review I imagine